ariadne/cardano/src/Ariadne/Wallet/Face.hs

```diff
@@ -100,6 +100,7 @@ data ConfirmationType
   = ConfirmMnemonic [Text]           -- ^ mnemonic
   | ConfirmRemove DB WalletSelection -- ^ walletDB, selection
   | ConfirmSend [ConfirmSendInfo]    -- ^ lists of outputs' info
+  | ConfirmDelUnknown Int            -- ^ number of unknown keys in keyfile
```

Comment:
Since this number is supposed to be small, I'd attach a list of root ids (probably as `Text`, because UI only needs `Text`) and display them instead of just a number.

Reply:
Do you really think, that it would be useful for a user?

Reply:
Well, I don't like that user has to decide whether they want to delete _some keys_ when they have no idea _which keys_.
Regarding usefulness:
1. Currently it's not so useful, because the only case when user sees root ID is when they select a wallet in TUI. However, if you have two wallets with money and they are displayed in TUI and the message propose to remove some key, you can check whether corresponding root ID is one of these two wallets. If it is, you will not delete it (and it means we have a bug). If it's something different (not one of two wallets you care about), you will be sure that you can safely delete it. I think we should display IDs in GUI as well, created https://issues.serokell.io/issue/AD-486
2. I think after AD-294 and AD-168 (which I want to be done soon) wallet ID will always be passed explicitly to backend commands, so one will be able to lookup ID in history (and maybe discover that earlier they removed it).

Reply:
Oh, ok. If a user will be able to see root ID in UI, then it will be definitely useful

Reply:
Yes, it's already possible in TUI and will be added to GUI later.